pkgs/development/node-packages/node-packages-v10.json

```diff
@@ -6,4 +6,5 @@
 , "node-gyp-build"
 , "node-pre-gyp"
 , "pnpm"
+, "ttf2eot"
```

Comment:
Please add it to `node-packages-v8.json` instead.

Reply:
thanks for the review. Updated.